mdflynn/game-sleuth
# Feature/nav

## Is this a fix or a feature?
This is a feature

## What is the change?
- Adds nav component with home button
- Adds unit test in Nav file
- Adds integration test for Nav in App

## What does it fix?
No fix this PR

## Where should the reviewer start?
Review the entire Nav.tsx and Nav.test.tsx file.

## How should this be tested?
Run `npm start` and navigate to localhost:3000 to see site functionality
Run `npm test` to see passing test suite.

## Relevant Issues
#11
